front/app/modules/commercial/user_custom_fields/services/__mocks__/userCustomFIeldsStats.ts

```diff
@@ -11,16 +11,19 @@ export const mockUsersByRegFields = {
   },
```

Comment:
Not your mistake, but I noticed there's a typo in the file name (userCustomF**I**eldsStats), would be nice to change :)